lrrbot/commands/quote.py

```diff
@@ -39,13 +39,14 @@ def format_quote(tag, qid, quote, name, date, context):
 		quote_msg += " [{date!s}]".format(date=date)
 	return quote_msg
 
-@bot.command("quote(?: (?:(\d+)|(.+)))?")
+@bot.command("quote\s*(?:(game|show)\s+(.+)|(?:(\d+)|(.+)))?")
```

Comment:
Spaces in the regular expression get replaced with `(?:\s+)`. So `'\s*'` should be `' ?'` and `'\s+'` should be `' '`. This regex also allows `!quotegame Dark Souls`. Is this intended?